linux_os/guide/services/ssh/ssh_server/sshd_include_crypto_policy/oval/shared.xml

```diff
@@ -25,8 +25,8 @@
     </ind:textfilecontent54_test>
 
     <ind:textfilecontent54_object id="obj_{{{ rule_id }}}_include_sshd_include_system_crypto" version="1">
-        <ind:filepath operation="pattern match">/etc/ssh/sshd_config</ind:filepath>
```

Comment:
this checks only files in the dropin directory... I think the pattern should consider also the file /etc/ssh/sshd_config

Reply:
Fixed. Test added.